src/org/labkey/test/tests/nab/NabAssayTest.java

```diff
@@ -803,6 +804,12 @@ private void verifyRunDetails()
         assayHelper.verifyDataIdentifiers(AssayImportOptions.VisitResolverType.SpecimenIDParticipantVisit, "D");
     }
 
+    private WebElement waitForNabGraph()
+    {
+        return shortWait().until(ExpectedConditions
+                .visibilityOfElementLocated(Locator.tagWithAttribute("img", "alt", "Neutralization Graph")));
```

Comment:
I'm guessing that since this is in the NabAssayTest the alt value will always be "Neutralization Graph". If not would it make sense to pass in the value for the alt value as a parameter?